playbooks/build-oc-images.yml

```diff
@@ -0,0 +1,51 @@
+---
```

Comment:
Please add a comment here (and in commit message) explaining that this playbook is not exposed via IR interface (ie `ir-installer`) but is rather intended for advance usage via `ansible-playbook` command